src/pe_source/data/pe_db/db_query.py

```diff
@@ -315,3 +330,37 @@ def insert_sixgill_topCVEs(df):
         logging.info(error)
         conn.rollback()
     cursor.close()
+
+
+def insert_shodan_data(dataframe, table, thread, org_name, failed):
+    """Insert shodan data into db."""
```

Comment:
Capitalization, etc:
```suggestion
    """Insert Shodan data into database."""
```